File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/SpeechRequest.kt
package com.sdkwork.ai

data class SpeechRequest(
    val model: String? = null,
    val input: String? = null,
    val voice: String? = null,
    val speed: Double? = null,
    val extra: Map<String, Any>? = null,
    val responseFormat: String? = null
)

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/speech_request.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class SpeechRequest:
    """Speech request"""
    model: str = None
    input: str = None
    voice: str = None
    speed: float = None
    extra: Dict[str, Any] = None
    response_format: str = None
